cfme/tests/generic_objects/test_instances.py

```diff
@@ -42,6 +45,58 @@ def tags(request, appliance, categories):
     return rest_gen_data.tags(request, appliance, categories)
 
 
+@pytest.fixture
+def textbox_dialog(appliance):
+    element_data = {
+        'element_information': {
+            'ele_label': "ele_" + fauxfactory.gen_alphanumeric(),
+            'ele_name': fauxfactory.gen_alphanumeric(),
+            'ele_desc': fauxfactory.gen_alphanumeric(),
+            'choose_type': "Text Box"
+        },
+        'options': {
+            'field_required': True
+        }
+    }
+    with appliance.context.use(ViaUI):
+        service_dialog = appliance.collections.service_dialogs.create(
+            label="test-dialog-{}".format(fauxfactory.gen_alpha()),
+            description="testing dialog"
+        )
+        tab = service_dialog.tabs.instantiate("New tab")
+        box = tab.boxes.create(
+            box_label="box_{}".format(fauxfactory.gen_alpha()), box_desc="my box"
+        )
+        box.elements.create(element_data=[element_data])
+
+    # # also create a view class for this dialog after it's accessed via a button
+    # class TextBoxDialogView(TextInputDialogView):
+    #     text_box = TextInput(locator=".//input")
+    #
+    #     @property
+    #     def is_displayed(self):
+    #         return self.submit.is_displayed and self.text_box.is_displayed
+
+    yield service_dialog
```

Comment:
Do we want to delete this as teardown?

Reply:
woops, actually meant to delete this fixture as it is no longer used